ndex2/client.py

```diff
@@ -1103,6 +1658,21 @@ def get_network_set(self, set_id):
     def add_networks_to_networkset(self, set_id, networks):
         """
         Add networks to a network set.  User must have visibility of all networks being added
+	
+	.. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		set_id = 'INSERT SET UUID HERE'
+
+		#the following codes return all the network ids in my account as a list
+		#then add the networks in the list to the set
+		my_networks = my_ndex.get_network_ids_for_user("ENTER YOUR USER")
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client